pytest/tests/mocknet/load_test_spoon.py

```diff
@@ -197,14 +195,14 @@ def check_slow_blocks(initial_metrics, final_metrics):
     test_passed = True
 
     script, deploy_time, test_timeout = (None, None, None)
-    if args.script == 'skyward':
-        script = 'load_testing_add_and_delete_helper.py'
-        deploy_time = load_testing_add_and_delete_helper.CONTRACT_DEPLOY_TIME
-        test_timeout = load_testing_add_and_delete_helper.TEST_TIMEOUT
-    elif args.script == 'add_and_delete':
+    # The flag is no longer needed but is kept for backwards-compatibility.
+    if args.script == 'add_and_delete':
```

Comment:
Is there any automation that depends on it? 
- no -> let's remove it
- yes -> let's create a small follow up task to remove it

Reply:
Cleaned up a little bit